recommendations/choices.py

```diff
@@ -226,217 +226,250 @@
 
 TRIGGERS = [
     ("", "No Triggers"),
-    ("animals are abused", "Animals Are Abused"),
-    ("a cat dies", "A Cat Dies"),
-    ("an animal dies", "An Animal Dies"),
-    ("a pet dies", "A Pet Dies"),
-    ("animals were harmed in the making", "Animals Were Harmed in the Making"),
-    ("there's dog fighting", "There's Dog Fighting"),
-    ("there's a dead animal", "There's a Dead Animal"),
-    ("a horse dies", "A Horse Dies"),
-    ("an animal is sad", "An Animal Is Sad"),
-    ("rabbits are harmed", "Rabbits Are Harmed"),
-    ("there are spiders", "There Are Spiders"),
-    ("there are bugs", "There Are Bugs"),
-    ("a dragon dies", "A Dragon Dies"),
-    ("there are snakes", "There Are Snakes"),
-    ("there are sharks", "There Are Sharks"),
+    (
+        [
+            "animals are abused",
+            "an animal dies",
+            "animals were harmed in the making",
+            "there's dog fighting",
+            "there's a dead animal",
+            "a pet dies",
+            "there's bestiality",
+        ],
+        "Animal Harm",
+    ),
+    (
+        [
+            "a child is abandoned by a parent",
+            "there's child abuse",
+            "there's abusive parents",
+            "there's pedophilia",
+            "a minor is sexualized",
+            "an infant is abducted",
+        ],
+        "Child Abuse",
+    ),
+    (
+        [
+            "there's addiction",
+            "alcohol abuse",
+            "someone uses drugs",
+            "someone overdoses",
+        ],
+        "Drug Usage / Abuse",
+    ),
+    (
+        [
+            "Autism specific abuse",
+            "there's domestic violence",
+            "someone cheats",
+            "there's bisexual cheating",
```

Comment:
I would move this to the LGBT section as, it's a harmful trope against bisexual people